lib/child-writer/index.js

```diff
@@ -126,8 +126,28 @@ exports.write = (image, drive, options) => {
    * }));
    */
   const bridgeRobotMessage = (message) => {
+    const ROBOT_COMMAND_LOG = 'log';
```

Comment:
Only the log string is used multiple times, which was the reason it was extracted as a constant. I'll add the error one just for symmetry purposes.